fe/fe-core/src/main/java/org/apache/doris/nereids/util/TypeCoercionUtils.java

```diff
@@ -823,6 +823,10 @@ public static Expression processComparisonPredicate(ComparisonPredicate comparis
         Expression left = comparisonPredicate.left();
         Expression right = comparisonPredicate.right();
 
+        if (left.getDataType().isBitmapType() || right.getDataType().isBitmapType()) {
+            throw new AnalysisException("Unsupported bitmap type in expression: " + comparisonPredicate.toSql());
```

Comment:
could not check here, because we need to support a in bitmap subquery. maybe we do not need do any change for nereids. Nereids comparison predicate do not accept bitmap already.